config/initializers/sidekiq.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "../../lib/sidekiq/honeycomb_middleware"
```

Comment:
Without this, I got a `NameError` from the Sidekiq workers when starting the app locally with `overmind start -f Procfile.dev`.